.gitignore

```diff
@@ -13,3 +13,8 @@ profile
 *.moved-aside
 .DS_Store
 *.xccheckout
+# CocoaPods
+Podfile.lock
```

Comment:
You must **never** gitignore `Podfile.lock`.

Podfile.lock is the manifest that tell with versions of the pods have been installed the last time you `pod install`. Keeping it ensure that anyone making a `pod install` on their own machine will get the same version of pods that the ones you used the last time you checked your code.

This is important because if for some reason a third-party lib update (even a patch) breaks something, the code that works now will suddenly stops working later **even if there were no new commit in `OHHTTPStubs`**… just because one dependency did publish a breaking update, and as you didn't commit the `Podfile.lock` so you didn't **lock** your dependencies.

That's the way every dependency manager (CocoaPods, Carthage, etc) work and `Podfile.lock` files are **meant** to be committed onto GIT, otherwise there would be no guarantee that running `pod install` one day on one machine will install the same stuff that running `pod install` days later or on another machine. (If you want to update your dependencies later, that's what `pod outdated` and `pod update XXX` and `pod update` are for anyway)